packages/d3fc-webgl/src/buffers/baseAttributeBuilder.js

```diff
@@ -19,13 +20,17 @@ export default () => {
         const location = gl.getAttribLocation(program, name);
         gl.vertexAttribPointer(
             location,
-            base.size(),
-            base.type(),
-            base.normalized(),
-            base.stride(),
-            base.offset()
+            size,
+            type,
+            normalized,
+            stride,
+            offset
         );
         gl.enableVertexAttribArray(location);
+        if (verticesPerElement !== 1) {
```

Comment:
This and line 33 in programBuilder fixed the issue with Point, but it feels a bit hacky - I'll speak to you later because I've forgotten exactly what but think you had something else in mind @chrisprice?

Reply:
Please can you raise an issue to cover fixing this up? I think there might be an interaction between this and the change @jamesmporter made in #1414, so I'm keen to see what the combined version of these changes is.

Reply:
Done https://github.com/d3fc/d3fc/issues/1421